linux/Jenkinsfile

```diff
@@ -245,7 +245,7 @@ def buildAndTest(String DISTRO, String buildArch) {
     }
 }
 
-private void buildCLi(String DISTRO, String buildArch, String GPG_KEY = null) {
+def buildCLi(String DISTRO, String buildArch, String GPG_KEY = null) {
     def buildCLI = "./gradlew package${TYPE}${DISTRO} check${TYPE}${DISTRO} --parallel -PPRODUCT=${env.PRODUCT} -PPRODUCT_VERSION=${VERSION} -PARCH=${buildArch}"
     if (GPG_KEY) {
         buildCli += " -PGPG_KEY=${GPG_KEY}"
```

Comment:
Wrong variable case here: buildCli should be buildCLi

Reply:
thats on Gerd, but i will fix it for him

Reply:
@steelhead31 is testing it, and if it is good we can merge this PR first and get JRE publish asap.
After that you are free to do more polish work @ascheman , how you think?

Reply:
I suggest to use #605 (just let me do a local test on my private Jenkins first).

Reply:
Or @steelhead31 could test my PR already including GPG signing (but without publishing the binaries of course)?